cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -3,6 +3,7 @@ private import DataFlowUtil
 private import DataFlowDispatch
 private import FlowVar
 private import DataFlowImplConsistency
+import codeql.util.Unit
```

Comment:
I agree, but that ended me up in a terrible mess because of the various ways the various QL libraries define/import `Unit`, and also some of the shared files depending on the definition of `Unit`.

Reply:
Yeah, I'll try to do another pass other this is merged.